### ISSUE TITLE
Accessibility Audit: Org sign-up flow

### ISSUE THREAD

**Issue by mehansen** [CDCgov/prime-simplereport#5765]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the org sign-up user flow. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/sign-up`
* `/app/sign-up/identity-verification`

Use cases to test:
* user selects that their org is already using SimpleReport
* user selects that they are trying to view their results
* user signs up for SimpleReport but is in a state we don't support yet and are prompted to join the waitlist
* user goes through identity verification workflow
    * they pass the Experian check
    * they are prompted to call to validate their identity
* after org identity verified, cont user sign up (inc all forms of MFA)

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

**Comment by emyl3** [comment]
tickets created:
https://github.com/CDCgov/prime-simplereport/issues/5881
https://github.com/CDCgov/prime-simplereport/issues/5882
https://github.com/CDCgov/prime-simplereport/issues/5880
